components/components.css

```diff
@@ -0,0 +1,591 @@
+
+.alert {
+  border: 1px solid transparent;
+  padding: 0.5rem 1rem;
+  margin: 0.5rem 0;
+  border-radius: 0.5rem;
+  text-align: center;
+  width: 45%;
+  font-size: 1.1rem;
+}
+
+.alert-success {
+  background-color: #9ae6b4;
+}
+
+.alert-warning {
+  background-color: #fbd38d;
+}
+
+.alert-error {
+  background-color: #feb2b2;
+}
+
+/* Badges */
+
+.badge {
+  border: 1px solid transparent;
+  background-color: var(--gray);
+  padding: 0.2rem 0.4rem;
+  /* border-radius:0.2rem; */
+  border-top-right-radius: 1rem;
+  border-bottom-right-radius: 1rem;
+  color: white;
+  font-weight: bold;
+  font-size: 0.7rem;
+  text-transform: uppercase;
+  width: 30px;
+}
+
+.badge-icon {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 50px;
+  height: 50px;
+  /*
+    background: #dddddd;
+    border: none;
+    border-radius: 50%;
+    margin:0.5rem 0; */
+}
+
+.badge-icon:hover {
+  cursor: pointer;
+}
+
+.badge-icon:active {
+  background-color: #cccccc;
+}
+
+.icon {
+  width: 35px;
+  height: 35px;
+}
+
+.badge-top {
+  position: absolute;
+  border: 2px solid white;
+  top: 0px;
+  right: 3px;
+  width: 20px;
+  height: 20px;
+  background: red;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  border-radius: 50%;
+  font-size: small;
+}
+
+.badge-avatar {
+  position: relative;
+  display: flex;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  justify-content: center;
+  align-items: center;
+  margin: 0 1rem;
+}
+
+.badge-bottom {
+  position: absolute;
+  border: 2px solid white;
+  bottom: -2px;
+  right: 1px;
+  width: 11px;
+  height: 11px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  border-radius: 50%;
+}
+
+.online {
+  background-color: rgb(4, 224, 4);
+}
+
+.offline {
+  background-color: rgb(224, 4, 4);
+}
+
+.btn {
+  border: none;
+  padding: 0.5rem 1rem;
+  cursor: pointer;
+  margin: 0.5rem;
+}
+
+.btn-sm {
+  width: 5rem;
+  word-wrap: break-word;
+}
+
+.btn-md {
+  width: 7rem;
+  word-wrap: break-word;
+}
+
+.btn-lg {
+  width: 9rem;
+  word-wrap: break-word;
+}
+
+.btn:hover {
+  /* border: 1px solid black; */
+  opacity: 0.9;
+}
+
+.btn:active {
+  transform: scale(0.98);
+}
+
+.btn-primary {
+  background-color: var(--primary-color);
+  border-radius: 0.25rem;
+}
+
+.btn-secondary {
+  background-color: var(--gray);
+  border-radius: 0.25rem;
+}
+
+.icon1 {
+  width: 15px;
+  height: 15px;
+}
+
+.btn-icon {
+  text-align: center;
+  background-color: var(--off-white);
+  border: 1px solid var(--dark-gray);
+  border-radius: 0.25rem;
+  padding: 0.5rem 1rem;
+}
+
+.float-action {
+  position: relative;
+}
+
+.btn-float {
+  position: fixed;
+  width: 40px;
+  height: 40px;
+  border-radius: 50%;
+  bottom: 0.5rem;
+  right: 1rem;
+  font-size: x-large;
+  text-align: center;
+  border: none;
+  outline: none;
+  background-color: var(--dark-gray);
+  cursor: pointer;
+}

```

Comment:
you can change the outline none to any specific colored outline, but avoid to make it none.

Reply:
Removed the ```outline: none``` from the class.

Reply:
Removed the ```outline: none``` property from the class.